Toxygates/src/panomicon/ScalatraJSONServlet.scala

```diff
@@ -66,23 +68,27 @@ class ScalatraJSONServlet(scontext: ServletContext) extends ScalatraServlet
     }
   }
 
-  def isDataVisible(data: Dataset, userKey: String) = {
```

Comment:
This is the deprecated "user data" mechanism from Toxygates that gives each user a unique hexadecimal ID; I don't believe we should keep that in the Angular application, and hardly anyone has been using it.